### PR TITLE
Product and shopping list view

### PR DESCRIPTION
**Pull request by abbytran1996**
Fixed some code recommended by Eric

### CODE REVIEW

File: src/com/tigersale/userInterface/customerUser/ListOfProductView.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            {
+                choice = scanner.nextInt();
+            } catch (InputMismatchException e)
+            {
+                System.out.println("Please type in an integer corresponding to your preferred option.");
+                scanner.next();
+                continue;
+            }
+            //Invalid option:
+            if (choice > 1)
```
Comment (ermammoser) [inline src/com/tigersale/userInterface/customerUser/ListOfProductView.java:129]: Use switch statement here.  You can do 
switch(choice)
{
case 0:
...
break;
case 1:
...
break;
default:
// this is your error case or "choice > 1"
break;

File: src/com/tigersale/userInterface/customerUser/ListOfProductView.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                try {
+                    choice = scanner.nextInt();
+                } catch (InputMismatchException e) {
+                    System.out.println("Please type in an integer corresponding to your preferred option.");
+                    scanner.next();
+                    continue;
+                }
+
+                //Invalid option
+                if (choice > (size + 1))
```
Comment (ermammoser) [inline src/com/tigersale/userInterface/customerUser/ListOfProductView.java:76]: Use switch statement here. You can do
switch(choice)
{
case 0:
...
break;
case 1:
...
break;
default:
// this is your error case or "choice > 1"
break;

**Comment by ermammoser** [comment]
Besides the comments I made about using switch statements over if statements, it looks good.  The only thing I want to verify is, have you run your code and stepped through all of the different cases?

**Comment by abbytran1996** [comment]
Yes I have tested several times before pushing. And thank you I will improved the my code like you have suggested!

**Comment by ermammoser** [comment]
Awesome, thanks.  So all you have to do is add, commit, and push your branch after your changes and this pull request automatically updates.  It is also nice because I can see all new changes from that commit alone

**Comment by abbytran1996** [comment]
Ok got it. Thank you for your advice, helped me learn a lot!

**Pull request merged**